src/caret_analyze/value_objects/callback.py

```diff
@@ -660,17 +701,8 @@ def service_name(self) -> str | None:
         return self._service_name
 
     @property
-    def publish_topic_names(self) -> tuple[str, ...] | None:
-        """
-        Get publisher topic names.
-
-        Returns
-        -------
-        tuple[str, ...] | None
-            Topic names which the callback publishes.
-
-        """
-        return self._publish_topic_names
+    def publish_topics(self) -> tuple[tuple[PublishTopicInfoValue, ...] | None]:
+        return self._publish_topics
```

Comment:
please add docstring.
And I think the type annotation for this line is `tuple[PublishTopicInfoValue, ...] | None`

Reply:
I think so too, but the following error occurs only in test_mypy, so the code is like this.
```
src/caret_analyze/value_objects/callback.py:729:16: error: Incompatible return value type (got "tuple[tuple[PublishTopicInfoValue, ...] | None]", expected "tuple[PublishTopicInfoValue, ...] | None")  [return-value]
```

Reply:
[This suggestion](https://github.com/tier4/caret_analyze/pull/460#discussion_r1545900729) appears to be related.

Reply:
I fixed